packages/backend-auth/src/lambda/reference_auth_initializer.ts

```diff
@@ -1,3 +1,5 @@
+// This executes within the custom resource lambda and errors get handled by the provider framework
+/* eslint-disable amplify-backend-rules/prefer-amplify-errors */
```

Comment:
For Lambda runtime sources we'd rather create local `.eslintrc.json` in `lambda` directory and put suppression there.

See example here https://github.com/aws-amplify/amplify-backend/blob/main/packages/backend/src/engine/branch-linker/lambda/.eslintrc.json .

Reply:
Updated